gpustack/config/config.py

```diff
@@ -136,6 +136,7 @@ class Config(BaseSettings):
     gateway_concurrency: int = 16
     service_discovery_name: Optional[str] = None
     namespace: Optional[str] = None
+    dockerhub_reachable: Optional[bool] = None
```

Comment:
Putting it here doesn't seem appropriate. Can we have a dedicated global variable for it?